W-15612083-Run-VALE-on-latest-Studio-repo-CP (#524)
mulesoft/docs-studio: modules/ROOT/pages/add-another-output-transform-studio-task.adoc

:page-aliases: 6.5.x@studio::add-another-output-transform-studio-task.adoc

> A single Transform Message component can give shape to several different parts of the output (example: the payload, a flow variable, a property). You must define each different output part in a separate DataWeave code, written out in a separate tab of the Transform pane. For example: if one tab defines the payload, and another an outbound property, these are both parts of the same output Mule message.

Reviewer: ```suggestion
A single Transform Message component can shape different output parts, such as the payload, a flow variable, or a property. You must define each output part in a separate DataWeave code, written out in a separate tab of the Transform pane. For example: if one tab defines the payload, and another an outbound property, both are parts of the same output Mule message.
```

Author: OK! Thanks!
